.github/workflows/release.yml

```
@@ -11,6 +11,9 @@ jobs:
   release:
     name: Release
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
```

```suggestion
      pull-requests: write
      id-token: write
```

We may need to add `id-token` to the permissions. Just based on the [instructions here](https://docs.npmjs.com/generating-provenance-statements#publishing-packages-with-provenance-via-github-actions) and following an other open source example ([sveltejs/kit](https://github.com/sveltejs/kit/blob/main/.github/workflows/release.yml#L15))